docs/root/version_history/current.rst

```diff
@@ -12,6 +12,7 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* access_log: added new access_log command operator ``%ENVIRONMENT(X):Z%``.
```

Comment:
Emm...is this a new feature? feel like this isn't a behavior change.

Reply:
Right, thanks for calling it out. 🌷